### ISSUE TITLE
Fix failing CI on building docs

### ISSUE THREAD

**Issue by JamesxX** [typst-community/valkyrie#16]
CI automatically builds the docs, which depend on a locally installed unreleased version of mantis. This issue could be resolved by either not requiring that CI successfully compiles the docs, or by some means installing this package during CI.

**Comment by tingerrr** [comment]
It's also a little problematic for contributors who wish to view their changes to the docs.

I would like to use [utpm](https://github.com/Thumuss/utpm) for management of pre-release versions of Typst packages, but I don't know if it's mature enough yet. The last commit was 3 months ago and when I tried it, it had some weird bugs.

Perhaps I could flesh utpm out to add the necessary features and fix the bugs I encountered with it. I don't think git packages are expected to hit anytime soon in Typst itself, so this may be necessary in the long run anyway.
